.github/workflows/main.yml

```diff
@@ -26,10 +26,6 @@ jobs:
           - Squeak64-5.3
           - Squeak64-5.2
           - Squeak64-5.1
-          - Squeak32-trunk
-          - Squeak32-6.0
-          - Squeak32-4.5
-          - Pharo64-stable
```

Comment:
Why remove `Pharo64-stable` here?

Reply:
1. Because that's the failing build
2. Because Pharo64-11 is a separate build, already in the list

I'm not saying this is a good suggestion :)